cocos/asset/assets/asset.jsb.ts

```diff
@@ -22,11 +22,12 @@
  THE SOFTWARE.
 */
 
-import { cclegacy, js, _decorator, path, jsbUtils, CallbacksInvoker, applyMixins } from '../../core';
+import { cclegacy, js, _decorator, path, CallbacksInvoker, applyMixins } from '../../core';
 import { getUrlWithUuid } from '../asset-manager/helper';
 import { patch_cc_Asset } from '../../native-binding/decorators';
 import type { Asset as JsbAsset } from './asset';
 import type { AssetManager } from '../asset-manager';
+import { ExtraEventMethods } from '../../core/utils/jsb-utils';
```

Comment:
After doing this, we can not make core as a module in future.

Reply:
Let's estimate that in the future. 
In fact, importing classes from the path is good for `treeshaking`.
If we export  `ExtraEventMethods` from 'core' and import it from 'core' here, `jsb-utils` will not be able to be cut by `rollup treeshaking`.